content/analyze/visualization-components/visualization-components.md

```diff
@@ -10,3 +10,15 @@ linked:
     - ./igv.md
 ---
 
+
+## Visualization Packages
+The HCA Visualization Package Registry lists JavaScript modules that enable analysis through interactive data exploration on the web.  
+
+ These solutions are not built by the HCA DCP implementation team but are solutions we foster and support. 
```

Comment:
A bit shorter and less redundant, while preserving Tim's original intent to distinguish:

```suggestion
These solutions are built by third parties, and fostered by the HCA DCP.
```